pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -293,6 +293,9 @@ in (buildEnv {
   # caches for it.
   ''
     if [[ -e "$out/bin/mtxrun" ]]; then
```

Comment:
I suppose it's also time to check whether `bin.context` is in the list of packages, as opposed to testing the file on disk.

Reply:
I suppose `pname = "context" && tlType = "bin"` would be the 'correct' test. This would detect if the `mtxrun` symlink is missing. (In an ideal world, this should be a hook provided by the `context` package, but that would be quite a waste of code for just one package.)

(PS: this is all academic discussion, because texlive is almost impossible to customize beyond `texlive.override` anyway.)

Reply:
one a similar note, one could debate what the correct behaviour is when mtxrun (from bin.context) exists, but the neccessary support file mtxrun.lua from the context tlpkg does not. We could
- just not run the hook since mtxrun is broken
- fail the build as the user probably wants to use context if they included bin.context
- silently add context

The current version fails the build in this case, but maybe the first option would be better. But this also falls into "almost certainly no one will ever be impacted by this"-territory.